### PR TITLE
Bug/fix filter dropdown issue 164040683

### PR DESCRIPTION
**Pull request by andela-fojuri**
## What does this PR do?
- Fixes the filter dropdown auto closing when a filter is selected

## Description of Task to be completed?
The opened filter dropdown(s) close automatically when a filter is selected and only the first dropdown remains open.

## What are the relevant pivotal tracker stories?
[#164040683](https://www.pivotaltracker.com/story/show/164040683)

## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
- [x] Raise PR
- [x] Test
- [ ] Documentation
## Screenshots

### CODE REVIEW

File: src/components/common/Filter/FilterComponent.jsx [hunk 1]
```diff
@@ -50,10 +37,11 @@ class FilterComponent extends React.Component {
           />
           <Accordion.Content active={activeIndex === index}>
             <Form className="filter-form">
-              {
-                option.content.map((opt) => {
+              <Form.Group grouped>
```
Comment (Arthur236) [inline src/components/common/Filter/FilterComponent.jsx]: The addition of this has messed with the styling that we had before. The filters look very squeezed now. kindly revert to the original

File: src/components/common/Filter/FilterComponent.jsx [hunk 2]
```diff
@@ -62,21 +50,21 @@ class FilterComponent extends React.Component {
                     if (selectedOptions[0] === false) {
                       selectedOptions[0] = 'UnVerified';
                     }
-                    return selectedOptions.includes((label).toString());
+                    return selectedOptions.includes(label.toString());
                   };
 
                   return (
-                    <CheckboxComponent
+                    <Form.Checkbox
```
Comment (Arthur236) [inline src/components/common/Filter/FilterComponent.jsx]: Why not use the CheckboxComponent instead?

**Comment by andela-fojuri** [comment]
Comments addressed @Arthur236

**Comment by Arthur236** [comment]
Awesome.

Though I've noticed that if you select a filter that you have to scroll to get to, it always resets to the first one. Could you look into this as well?

**Pull request merged**